src/main/java/com/mineinabyss/looty/ecs/systems/CooldownDisplaySystem.kt

```diff
@@ -43,8 +44,10 @@ object CooldownDisplaySystem : TickingSystem(interval = INTERVAL) {
                     repeat(squaresLeft) {
                         append(displayChar)
                     }
-                    append(ChatColor.GRAY)
-                    append(" [$timeLeft]")
+                    if (timeLeft.toDouble(DurationUnit.MILLISECONDS) < 250) append(
```

Comment:
Again, reuse the INTERVAL constant here, you'd need to say < INTERVAL * 1000 / 20 though to convert from ticks to milliseconds

Reply:
Is there any reason that you suggest 1000/20 instead of just * 50. I understand if it's to make it easier to see 20 ticks every second

Reply:
yeah feel free to simplify it here. I'd personally go for the more explicit conversion since it's self-documenting and there's no performacne impact at all.